bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxHandler.java

```diff
@@ -125,4 +129,37 @@ public void handleConfigurationUpdate(Map<String, Object> configurationParameter
             super.handleConfigurationUpdate(configurationParameters);
         }
     }
+
+    /**
+     * Remove previously statically created vane channel if the device does not support it. Or log a warning if it does
+     * support a vane and the respective channel is missing.
+     *
+     * @param bridgeHandler the calling bridge handler.
+     * @throws IllegalStateException if something went wrong.
+     */
+    public void updateDynamicChannels(VeluxBridgeHandler bridgeHandler) throws IllegalStateException {
+        // roller shutters are the only things with a previously statically created vane channel
+        if (!VeluxBindingConstants.THING_TYPE_VELUX_ROLLERSHUTTER.equals(thing.getThingTypeUID())) {
+            return;
+        }
+
+        String id = VeluxBindingConstants.CHANNEL_VANE_POSITION;
+        ChannelUID uid = new ChannelUID(thing.getUID(), id);
+        Thing2VeluxActuator actuator = new Thing2VeluxActuator(bridgeHandler, uid);
+        VeluxProduct product = bridgeHandler.existingProducts().get(actuator.getProductBridgeIndex());
+
+        if (product.equals(VeluxProduct.UNKNOWN)) {
+            throw new IllegalStateException("initializeVanePositionChannel(): Product unknown in the bridge");
```

Comment:
Minor: maybe you would like to change "initializeVanePositionChannel()" into "updateDynamicChannels()" ?

Reply:
Oops. Done.